io.github.cboxdoerfer.FSearch.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.cboxdoerfer.FSearch
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: fsearch
+build-options:
+    build_args:
+        - --share=network
```

Comment:
this is not allowed. There is no network during the build phase.

Reply:
Thx, fixed.